_spinnaker/configure_kayenta.md

```diff
@@ -0,0 +1,253 @@
+---
+layout: post
+title: Automated Canary Deployments
+order: 50
+---
+# Automating Deployments with Kayenta
+{:.no_toc}
+* This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
+{:toc}
+
+## Configure Kayenta
+
+The open source Spinnaker documentation has a good overview of how to 
+configure Kayenta via Halyard at
+[https://www.spinnaker.io/setup/canary/](https://www.spinnaker.io/setup/canary/)
+
+## Enable Canarying in Application
+
+Before you can use Canary stages in your application's pipelines, you'll need
+to enable the Canary feature in the application's config:
+
+![image](https://dha4w82d62smt.cloudfront.net/items/110k1y0g0l2a2s2w452C/%5Be5cd4808aad349ab8c521f7e97100f50%5D_Image+2018-04-17+at+9.25.33+AM.png)
+
+Make sure you remember to save the change.  We recommend you refresh the page
+in your browser after enabling the feature before continuing.
+
+Your "Pipelines" menu item should now be "Delivery"; if you click on it,
```

Comment:
I sometimes I like to do something like this to make it easier to look for the key words like `Pipelines` and `Delivery` so its easy to pick out.
>>>Your `Pipelines` menu item should now be `Delivery`; if you click on it,
you'll see `Pipelines` is a submenu item along with `Canary Configs` and
`Canary Reports`.

Reply:
or maybe **bold** them?